### PR TITLE
feat: [P4PU-672] removed transactions api

### PR DESCRIPTION
**Pull request by oleksiybozhykntt**
removed transactions api

### CODE REVIEW

File: .devops/soak-tests-pipeline.yml [hunk 1]
```diff
@@ -32,7 +32,6 @@ parameters:
       - getNoticeReceipt
       - getPaymentNotices
       - getZendeskAssistanceToken
-      - getTransactions
```
Comment (antonioT90) [inline .devops/soak-tests-pipeline.yml:35]: let's add also 'runAll', 'runAllAuth', etc...

**Review by antonioT90 — CHANGES_REQUESTED** [review]

File: .devops/soak-tests-pipeline.yml (hunk 1 above)
Comment (oleksiybozhykntt) [reply .devops/soak-tests-pipeline.yml:35]: added

File: .devops/soak-tests-pipeline.yml [hunk 2]
```diff
@@ -32,7 +32,11 @@ parameters:
       - getNoticeReceipt
       - getPaymentNotices
       - getZendeskAssistanceToken
-      - getTransactions
+      - runAll
+      - runAllAuth
+      - runAllNotice
+      - runAllPaymentNotice
+      - runAllTokenAssistance
```
Comment (antonioT90) [inline .devops/soak-tests-pipeline.yml]: if we rearrange this list in order to have:
* runAll as first
* runAll-<domain> before all domain related tests?

File: .devops/soak-tests-pipeline.yml [hunk 3]
```diff
@@ -75,9 +79,6 @@ parameters:
 
 variables:
   # Comma separated list of tests to execute based on the SCRIPT parameter (if some test require the execution of more than one test, eg data preparation)
-  - name: k6Test_getTransactions
-    value: transactions/getTransactions
-
   - name: k6Test_getUserInfo
     value: auth/getUserInfo
```
Comment (antonioT90) [inline .devops/soak-tests-pipeline.yml]: for each `runAll` you have to define here the folder to run

File: .devops/soak-tests-pipeline.yml [hunk 4]
```diff
@@ -104,7 +105,18 @@ variables:
 
   # Resolved list of k6 script to execute
   - name: SCRIPTS_TO_EXECUTE
-    value: ${{ variables[format('k6Test_{0}', parameters.SCRIPT)] }}
+  - ${{ if eq(parameters.SCRIPT, 'runAll') }}:
```
Comment (antonioT90) [inline .devops/soak-tests-pipeline.yml]: this is not required!

**Review by antonioT90 — CHANGES_REQUESTED** [review]

**Review by antonioT90 — APPROVED** [review]

**Pull request merged**